### ISSUE TITLE
Causes BOP to fail

### ISSUE THREAD

**Issue by Maccool** [Markus1002/Incubation#6]
I added this mod and tried to open an existing game and got the error message in the screen shot: 

Error loading mods
1 error encountered during loading
Biomes O' Plenty (biomesoplenty) encountered an error during the common_setup event phase
null

When I removed Incubation mod Forge was able to complete successfully. forge 1.14.4-forge-29.2.23, BiomesOPlenty-1.14.4-9.0.0.262, incubation-1.0.1-1.14.4

![2020-08-31-172509_1920x1080_scrot](https://user-images.githubusercontent.com/5578756/91775023-bd8a8f00-ebaf-11ea-87ba-4a7618cfb1e3.png)

Thanks

**Comment by Vaelzan** [comment]
Technically this is due to a bug existing in both mods, and it is in 1.14, 1.15, and 1.16.

It's caused by Incubation and Biomes o' Plenty both registering Compostables and Flammables during the common setup phase, which can occasionally cause a crash, since mod loading is processed in parallel and registering those things is *not* thread safe. The correct method is to register them using deferred registration. I've previously fixed this same issue in several Minecraft Abnormals mods, but noticed that a lot of mods have the problem (Quark too, frustratingly).

I'm not sure that you'll ever get a fix for 1.14.4, since not many mod developers support that version anymore.

I was personally going to fix the issue for 1.15.2 via a pull request since it is causing trouble in the Valhelsia 1.15.2 packs, but that branch doesn't even exist, with the port to 1.16.1 overwriting it. :(

**Comment by Maccool** [comment]
I have since moved on to MC 1.16.3 (enough mods I wanted had caught up to this), but thanks for the insight.
